README.md

```diff
@@ -1 +1,164 @@
-# payment-button-drupal-plugin
+Payment Button Drupal Plugin
+==============
+
```

Comment:
Do add following section
- support 
- License
- Change log

Reply:
Support and Change log are added in readme file
for License we have other file with name of LICENSE.txt